src/main.rs

```diff
@@ -31,10 +31,18 @@ use tee::TeeReader;
 use tempdir::TempDir;
 use xz2::read::XzDecoder;
 
-mod git;
 mod least_satisfying;
+mod repo_access;
 
 use crate::least_satisfying::{least_satisfying, Satisfies};
+use crate::repo_access::{AccessViaGithub, AccessViaLocalGit, RustRepositoryAccessor};
+
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
Thoughts about impl Display on the Commit struct?

Reply:
(lets table this to issue #64)